src/humancellatlas/data/metadata/helpers/schema_validation.py

```diff
@@ -37,7 +37,7 @@ def validate_json(self, file_json: JSON, file_name: str):
             schema_url = file_json['describedBy']
             raise RequirementError('Failed to parse schema JSON',
                                    file_name, schema_url) from e
-        self.validator.evolve(schema=schema)
+        self.validator = self.validator.evolve(schema=schema)
         try:
             self.validator.validate(file_json)
         except ValidationError as e:
```

Comment:
RequirementError is deprecated. Please enable deprecation inspection in PyCharm so are made aware of deprecations while touching code.